### DISCUSSION TITLE
Allow for custom notifications for only when a discussion is answered

### DISCUSSION THREAD

**Discussion by karlhorky** [community/community#8] [Discussions]
Discussions can generate a lot of notifications, which are in some cases not relevant for the user who has subscribed to the discussion.

For example, sometimes a user may want to just be notified only once there is a solution to a particular discussion (an "Answer" in GitHub terminology).

It would be nice to have the ability to set custom notification types on Discussions, so that a user would only receive a notification once a Discussion receives an Answer.

### Prior art

The custom notifications for issues and pull requests for:

- when a pull request is merged
- when a PR / issue is closed
- when a PR / issue is reopened

<img src=https://user-images.githubusercontent.com/1935696/96236587-87a53c80-0f9c-11eb-8697-0708ae231632.png alt="" width=370>

<img src=https://user-images.githubusercontent.com/1935696/96236749-b3c0bd80-0f9c-11eb-9b46-537ea8d2ec8f.png alt="" width=440>

**Comment by JakubKoralewski** [comment]
Also being able to subscribe only to responses to a certain comment by either clicking the triple dot vertical hamburger icon in the top right of this comment and selecting subscribe to replies would be nice.

**Reply by karlhorky** [reply]
Sounds like a nice addition!

Would you create a new discussion in `github/feedback` about this? And link it here? I would upvote!

**Reply by JakubKoralewski** [reply]
Sure. Here you go: https://github.com/github/feedback/discussions/3150

**Comment by evi-liu** [comment]
Thanks for the feedback, y'all! This is helpful as we iterate on our notifications experience. Are there any other customizations you'd desire besides `Answered`? How about being able to filter out to select categories?

**Reply by karlhorky** [reply]
@JakubKoralewski  mentioned being able to scope subscribes to a single comment thread: https://github.com/github/feedback/discussions/8#discussioncomment-274788

**Reply by karlhorky** [reply]
> How about being able to filter out to select categories?

What are "categories" in this context? What would be some examples? I'm not sure I understand what a category is related to a GitHub Discussion, but maybe I'm just not experienced enough here.

**Reply by JakubKoralewski** [reply]
@evi-liu You mean as an owner/watcher of the repository to only be notified of e.g. Q&A, but not General? That would be nice!

I'm more looking at it from the 3rd party perspective, i.e. I find an interesting discussion with a question that I'd like to know the answer to and I'm interested in solutions, sometimes not only the first one (one that is chosen as the answer) but also any future answers that may turn out to be even better thatn the accepted one. I know it's not very specific but I wanted to share my expectations of how I most often use Discussions.
